### PR TITLE
Crud Routes for Postgres

### PR DESCRIPTION
**Pull request by djian001**

### CODE REVIEW

File: server/sdcIndex.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+app.use(compression());
+app.use(express.static(path.join(__dirname, '..', '/public')));
+//app.use(bodyParser.json());
+//app.use(express.urlencoded({extended: true}));
+
+app.get('/', (req, res) => {
+  res.end();
+});
+
+app.get('/api/price/:bookId(\\d+)', (req, res) => {
```
Comment (knechtgese1) [inline server/sdcIndex.js:21]: Does the \\d+ restrict it to just grab digits?

File: database/methods/pgMethods.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      //console.log('FOUND IN PG', res.rows[0]);
+      return res.rows[0];
+    })
+    .catch((err) => {
+      console.log('error finding in PG', err)
+    })
+  ;
+}
+
+const createBook = async (data) => {
```
Comment (knechtgese1) [inline database/methods/pgMethods.js:39]: I think we needed some sort of update method, as well?

**Review by knechtgese1 — COMMENTED** [review]
Looks fine, although I think we needed an "update" function for both databases (since that one is the slowest -- the DB has to find the record, and then also write to it). Easy enough to do.

File: database/methods/pgMethods.js [hunk 3]
```diff
@@ -30,10 +30,34 @@ const createBook = async (data) => {
       console.log('Successfully Created new Book in pricing');
     })
     .catch((err) => {
-      console.log('error creating new book in pricing');
+      console.log('error creating new book in pricing', err);
+    })
+}
+
+const updatePrice = async (id, price) => {
+  let query = `UPDATE pricing SET price = ${price} WHERE book_id = ${id};`;
```
Comment (knechtgese1) [inline database/methods/pgMethods.js:51]: OK, I just need to actually read all of your code before opening my big fat mouth. :-D Looks fine!

**Review by knechtgese1 — COMMENTED** [review]
These look fine! I take back my comments about needing update and delete routes -- you have them here.

File: server/sdcIndex.js (hunk 1 above)
Comment (djian001) [reply server/sdcIndex.js:21]: this was part of the legacy code but from what i googled, it looks like its a regex for getting 1 or more digits.

**Pull request merged**